script/check

```diff
@@ -7,6 +7,9 @@ SCRIPTS=$(find script -type f | grep -vE "/(linting/|testing/|typing/|check$)")
 ./script/testing/test
 result=$?
 
+./script/testing/integration-test
+result=$((result | $?))
+
```

Comment:
This makes this script a lot more annoying to quickly use to check for linting and typing issues.

Reply:
Yeah, I wasn't sure about this. I found however that I was forgetting to run the other checks while working on the integration tests so having them all together seemed preferable. It should be easy enough to use the individual scripts if people want to run the other checks.

ffc7f01e8d55f65b76ceac31a3de737f5ca006c0 speeds up the integration tests; now reliably under 3s for me (as reported by unittest, obviously webots spin-up is outside that).

I guess we could add an option/environment variable here which turns off the integration tests if people want that? I do think the default should be everything though.

Another option (that I would encourage anyway) for faster feedback is configuring your editor to run the linters in the background & show squiggles.